registry/kusama.json

```diff
@@ -230,5 +230,13 @@
         "shortDescription": "Polkadot Moderator, Ambassador and full time Dev. I actively contribute to, advocate for, and guide the community's growth and progress.",
         "longDescription": "As a voting delegate at Kusama, I vote on proposals after deep understanding and discussions with community members. If a proposal is unclear, I abstain from voting. My aim is to ensure informed voting that benefits our community and drives progress.",
         "isOrganization": false
+    },
+    {
+        "address": "HyLisujX7Cr6D7xzb6qadFdedLt8hmArB6ZVGJ6xsCUHqmx",
+        "name": "metaspan",
+        "image": "https://raw.githubusercontent.com/metaspan/metaspan.io/main/ui3/public/image/globe.png",
+        "shortDescription": "Validating on Polkadot & Kusama. Actively supporting and contributing to the community via development, utils, education, and advocacy.",
+        "longDescription": "**metaspan** is a trusted network infrastructure provider and validator on Polkadot and Kusama.\nWe are a team of experienced engineers and developers with a passion for blockchain technology and the Polkadot ecosystem.\nWe are actively involved in the community and contribute to the ecosystem via development, utilities, education, and advocacy.\nWe are committed to the long-term success of Polkadot and Kusama and are excited to be a part of the journey.",
+        "isOrganization": true
     }
```

Comment:
```suggestion
    },
    {
        "address": "HyLisujX7Cr6D7xzb6qadFdedLt8hmArB6ZVGJ6xsCUHqmx",
        "name": "metaspan",
        "image": "https://raw.githubusercontent.com/metaspan/metaspan.io/main/ui3/public/image/globe.png",
        "shortDescription": "Validating on Polkadot & Kusama. Actively supporting and contributing to the community via development, utils, education, and advocacy.",
        "longDescription": "**metaspan** is a trusted network infrastructure provider and validator on Polkadot and Kusama.\n\nWe are a team of experienced engineers and developers with a passion for blockchain technology and the Polkadot ecosystem.\n\nWe are actively involved in the community and contribute to the ecosystem via development, utilities, education, and advocacy.\n\nWe are committed to the long-term success of Polkadot and Kusama and are excited to be a part of the journey.",
        "isOrganization": true
    }
```

Reply:
I suggest to improve a bit your formatting

Reply:
this one is outdated